src/mito2/src/request.rs

```diff
@@ -384,16 +382,16 @@ pub(crate) fn validate_proto_value(
 
 /// Oneshot output result sender.
 #[derive(Debug)]
-pub(crate) struct OutputTx(Sender<Result<Output>>);
```

Comment:
For the comment in PR description, it's mainly if we need the `AffectedRows` name in these type references.